bundles/org.openhab.binding.linky/README.md

```diff
@@ -1,35 +1,51 @@
 # Linky Binding
 
 This binding uses the API provided by Enedis to retrieve your energy consumption data.
-You need to create an Enedis account [here](https://espace-client-connexion.enedis.fr/auth/UI/Login?realm=particuliers) if you don't have one already.
+You need to create an Enedis account [here](https://mon-compte-client.enedis.fr/) if you don't have one already.
 
 Please ensure that you have accepted their conditions, and check that you can see graphs on the website.
 Especially, check hourly view/graph. Enedis may ask for permission the first time to start collecting hourly data.
+
 The binding will not provide these informations unless this step is ok.
 
+This new binding version is able to use multiple bridge to access the consumption data.
+You can use :
+
+- The EnedisWebBridge : this one will use the old Enedis API, base on the enedis web site to gather the data.
```

Comment:
Bridges names here have to be changed accordingly to last PR.

Reply:
fix in next commit